### DISCUSSION TITLE
New Medic Talent Tree

### DISCUSSION THREAD

**Discussion by Dailyyy** [FakeFishGames/Barotrauma#8453] [Ideas and suggestions]
Biohazard Talent Tree

Tier 1 choices:
Potent Poisons
Poisons you fire at enemies will apply 20% quicker and their effects will be 20% more severe. Also applies to aggressive humans (like bandits), and also crewmembers if you are a traitor.

Good Immunity System
Poisons that are inflicted upon you are 10% less potent and apply 10% slower.

Tier 2 choices:
Drug Storage
Maximum syringe gun chemical capacity is upped to 8. Poisons in syringe guns become 20% more effective.

Raptor Reaper
Raptor's Bane applies to mudraptors 60% quicker. Fertillizer used on Raptor Bane plants is 10% more effective and lasts 10% longer.

Tier 3 choices:
Syringe Machinegun
Syringe guns fire 50% faster. Does not apply to the Syringe Shotgun.

Even More Potent Poisons
Poisons you fire at enemies will now apply 40% quicker and their effects will be 30% more severe. Also applies to aggressive humans (like bandits), and also crewmembers if you are a traitor.

Tier 4 choices:
Biotoxins
Unlock recipe: Botulinum toxin
All toxins injected into you have a 10% chance to not apply their effects at all.

Back, you demon!
All enemies inside your submarine will have poison effects apply twice. Does not apply to Raptor's Bane.

Tier 5 choice:
Syringe Shotgun
Unlock recipe: Syringe Shotgun
Syringe gun ammo capacity is increased to 16. (Also applies to the Syringe Shotgun)
Syringe Shotgun would take only 1 syringe per shot and fire out 3 in a wide spread, each syringe applying 40% of the medicine fired's effects.

**Comment by someguythatplaysbarotrauma** [comment]
this sounds like a really cool talent tree that would make medics more useful besides being good at healing crewmembers, but what would the Botulinum toxin do, and what would you need to craft it? would it take one of every toxin, or would it only take a few toxins, OR would it take entirely new items to craft?

**Reply by Dailyyy** [reply]
reaaaally late response but most likely would paralyse the person being injected and also kill them over time
